TMDBApp/TMDBApp/Language.swift

```diff
@@ -0,0 +1,37 @@
+//
+//  Language.swift
+//  TMDBApp
+//
+//  Created by Richard Leh on 17/01/2017.
+//  Copyright © 2017 Richard Leh. All rights reserved.
+//
+
+import Foundation
+
+struct Language{
+    static let defaulUserKey = "defaulLanguage"
+    static let appResult = ["en-US", "pt-BR", "es-ES", "hr-HR", "ja-JP"]
```

Comment:
The advantage is that I don't need to call rawValue each time.
In this case I prefer array because If I want to add more languages I just put more a value on this array.